site/content/configuration/configuration-overview.md

```diff
@@ -160,41 +160,42 @@ nginx-agent
 {{<bootstrap-table "table table-responsive table-bordered">}}
```

Comment:
can you add a note under this title saying that the old prefix NMS has been deprecated, we will try automatically migrate any existing environment variables, to the new prefix NGINX_AGENT. You should use the NGINX_AGENT prefix going forward for all environmental variables.

Reply:
@ADubhlaoich to follow up with this warning so we can merge this PR